reactivesocket-discovery-eureka/src/main/java/io/reactivesocket/discovery/eureka/Eureka.java

```diff
@@ -51,6 +52,9 @@ public void subscribe(Subscriber<? super Collection<SocketAddress>> subscriber)
             private synchronized void pushChanges(Subscriber<? super List<SocketAddress>> subscriber) {
                 List<InstanceInfo> infos = client.getInstancesByVipAddress(vip, secure);
                 List<SocketAddress> socketAddresses = infos.stream()
+                    .filter(instanceInfo -> {
+                        return instanceInfo.getStatus() == InstanceStatus.UP;
```

Comment:
nittiest of nits: 

Can be simplified down to 1 line

`.filter(i -> i.getStatus() == InstanceStatus.UP)`